unicore/distribute/api/validators.py

```diff
@@ -32,3 +35,14 @@ def validate_schema(request):
     else:
         request.schema = schema.to_json()
         request.schema_data = data
+
+
+def repo_url_type_schema_validator(node, value):
+    valid_prefixes = ['git://', 'http://', 'https://', 'ssh://']
```

Comment:
I recall having issues pushing using anything other than `https` (I think). Wouldn't this be a problem here too?